LoopKit/AnalyticsService.swift

```diff
@@ -10,6 +10,9 @@ import Foundation
 
 public protocol AnalyticsService {
 
+    /// The unique identifier of this type of service.
+    var serviceIdentifier: String { get }
```

Comment:
This is part of the Service protocol as well.  Should any service conforming to `AnalyticsService` also conform to `Service`?